### PR TITLE
Endangered language totonac recipe

### PR DESCRIPTION
**Pull request by ftshijt**

### CODE REVIEW

**Comment by ftshijt** [comment]
The CI error will be fixed after #3839 is merged

File: egs2/totonac/asr1/RESULTS.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<!-- Generated by scripts/utils/show_asr_result.sh -->
+# RESULTS
+## Environments
+- date: `Fri Nov 12 02:34:12 EST 2021`
+- python version: `3.9.7 (default, Sep 16 2021, 13:09:58)  [GCC 7.5.0]`
+- espnet version: `espnet 0.10.4a1`
+- pytorch version: `pytorch 1.9.0`
```
Comment (sw005320) [inline egs2/totonac/asr1/RESULTS.md:7]: Can you upload a model?

Comment (ftshijt) [reply egs2/totonac/asr1/RESULTS.md:7]: Sure, I can submit a transformer model now but a bit more time to run for feature fusion, cause Dan changed the interface, so my current pretrained model may not fit into the release code.

Comment (DanBerrebbi) [reply egs2/totonac/asr1/RESULTS.md:7]: I can upload a model with the new config later this week if you want @ftshijt

Comment (ftshijt) [reply egs2/totonac/asr1/RESULTS.md:7]: Many thanks, that would be very helpful! If you want, you can use my prepared data folder to accelerate the procedure.

Comment (ftshijt) [reply egs2/totonac/asr1/RESULTS.md:7]: I will dm you.

**Pull request merged**